### PR TITLE
Allow staff to view private tests

### LINKED ISSUE #1430: Staff should be able to view private test results
When viewing a student's submission, results of private tests are not showing up.

### PR DESCRIPTION
**Pull request by caramelcat**
private_tests

Fixes #1430.

### CODE REVIEW

File: app/views/course/assessment/answer/programming/_test_cases.html.slim [hunk 1]
```diff
@@ -41,11 +41,14 @@ table.table.table-striped.table-hover
             = fa_icon 'check'.freeze
           - elsif test_case_result
             = fa_icon 'times'.freeze
-  - if can?(:grade, answer.answer) && !submission.attempting?
+  - if can?(:grade, answer.answer)
     tbody
       tr
         th colspan=5
           = t('.private')
+          span< class="bg-danger text-danger"
```
Comment (allenwq) [inline app/views/course/assessment/answer/programming/_test_cases.html.slim]: `span.bg-danger.text-danger` will do.

File: app/views/course/assessment/answer/programming/_test_cases.html.slim [hunk 2]
```diff
@@ -41,11 +41,14 @@ table.table.table-striped.table-hover
...
+            span class="glyphicon glyphicon-warning-sign" aria-hidden="true"
```
Comment (allenwq) [inline app/views/course/assessment/answer/programming/_test_cases.html.slim]: @caramelcat We use font-awesome for icons, can use `=fa 'icon-name'` to display the icon.

**Comment by benleong** [comment]
We need to improve the grammar of the message:

"You are able to view the private test cases because you are staff.
Students will not be able to see them."

**Comment by caramelcat** [comment]
**Staff view:**

staff




**Student view:**

student

**Pull request merged**